### PR TITLE
Chrome ext

### LINKED ISSUE #4: Create Chrome Extension

### PR DESCRIPTION
**Pull request by dumbbillyhardy**
closes #4

### CODE REVIEW

File: webclient/index.html [hunk 1]
```diff
@@ -124,7 +124,29 @@
...
+                });
+            }
+            let bitch = Number(window.localStorage.getItem("parseOrderBitch"));
+            function requestDom() {
+                bitch = (bitch === bitch ? bitch: 0) +1;
+                chrome.storage.local.set({parseOrderBitch: bitch});
+            }
+            document.getElementById("parse").addEventListener("click", requestDom);
         </script>
+        <button id="parse">Parse Dom? Wut?</button>
```
Comment (jonsmithers) [inline webclient/index.html]: back again?

**Review by jonsmithers — CHANGES_REQUESTED** [review]

File: webclient/index.html [hunk 2]
```diff
@@ -124,7 +124,29 @@
...
+                    let order = Order.fromJSON(JSON.parse(keyValueMap.order));
+                    table.order = order;
+                });
+            }
+            let bitch = Number(window.localStorage.getItem("parseOrderBitch"));
+            function requestDom() {
+                bitch = (bitch === bitch ? bitch: 0) +1;
+                chrome.storage.local.set({parseOrderBitch: bitch});
+            }
+            document.getElementById("parse").addEventListener("click", requestDom);
```
Comment (jonsmithers) [inline webclient/index.html]: this is the webclient index. It seems to me none of this should be here?

**Review by jonsmithers — CHANGES_REQUESTED** [review]

File: webclient/index.html (hunk 2 above)
Comment (dumbbillyhardy) [reply webclient/index.html]: good catch

File: webclient/index.html (hunk 1 above)
Comment (dumbbillyhardy) [reply webclient/index.html]: shouldn't be in index, but I couldn't get it to reliably notify the content script with just calling the function once

**Review by jonsmithers — APPROVED** [review]

**Pull request merged**